aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -966,7 +965,7 @@ Tensor& linalg_solve_out(const Tensor& input, const Tensor& other, Tensor& resul
   // Now check LAPACK/MAGMA error codes
   // batchCheckErrors(Tensor, char*) calls 'infos = infos.to(kCPU)'
   bool vector_case = linalg_solve_is_vector_rhs(input, other);
```

Comment:
Internal builds correctly complain this variable is now unused. Is that intentional?

Reply:
Yes, it's intentional. I should remove it.